Content.Server/Roles/NinjaRoleComponent.cs

```diff
@@ -31,10 +19,4 @@ public sealed partial class NinjaRoleComponent : AntagonistRoleComponent
     /// </summary>
     [DataField("spiderChargeDetonated")]
```

Comment:
this can be put on the objective too (but not target)

Reply:
i leave this for next PR

Reply:
why not its just a bool

Reply:
i want to move target to condition too

Reply:
you cant just move the field youd have to move the actual target finding into it too, then remove the target requirement component/system